examples/eqpt_config.json

```diff
@@ -0,0 +1,43 @@
+{     "Edfa":[{
+            "type_variety": "std_medium_gain",
+            "gain_flatmax": 25,
+            "gain_min": 15,
+            "p_max": 21,
+            "advanced_config_from_json": "std_medium_gain_advanced_config.json"
```

Comment:
Is there a distinction between "advanced config" and another type of config?

Ideally, we don't want one file pointing to other files. Ideally, we have a much flatter tree with the network config points to all of these files directly.

Reply:
not an easy one, I'm trying to make it as clear as possible, suggestions are welcome.
For now there is a "advanced_config_from_json": parameter to read the detailed nf and ripple  model of Ciena from a user input json file that is specific to each amplifier variety_type.
If this advanced_config_from_json is not present in the edfa variety type definition, then a more basic model is used that use a standard (=the same for all amplifiers) input json file.It is hidden in the code so the user doesn't have to worry about it. If wanted the user to be able to input quickly a new edfa variety type without any measurment, nor json file to provide.